src/gocept/pytestlayer/doctest.py

```diff
@@ -0,0 +1,31 @@
+import doctest
+
+
+class NoOpLayer(object):
```

Comment:
We are already Python 3 only with this package, so I would suggest: 
```suggestion
class NoOpLayer:
```